### PR TITLE
Add acontis EC-Master EtherCAT support to Beckhoff CX9020

### PR DESCRIPTION
**Pull request by PaulBussmann-acontis**
The Beckhoff CX9020 can run the acontis EC-Master, but needs the additional acontis atemsys Kernel module (GPL) as prerequisite. 

The changes of this pull request include all needed scripts to get a prepared CX9020 to run the acontis EC-Master package. 
The integration of the acontis EC-Master is clearly stated optional and not included by default.

Please integrate the changes in order to support EC-Master on CX9020.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,12 @@ cd CX9020/
 #=============
 make uboot
 
+#integrate acontis kernel extension atemsys from EC-Master SDK for emllCCAT support (optional):
+#==============================================================================================
+# The acontis EC-Master evaluation version can be requested at https://www.acontis.com/en/eceval.html
```
Comment (pbruenn) [inline README.md]: This is a real showstopper. Without a possibility to automatically test your contribution, I cannot merge this commit. As soon as your SDK is available for direct download, we can include that download link into your prepare script and I would be happy to merge another EtherCAT Master example into this repository.

File: README.md [hunk 2]
```diff
@@ -23,6 +23,12 @@ cd CX9020/
...
+export ACONTIS_EC_MASTER_SDK_PACKAGE=~/Downloads/EC-Master-V3.0-Linux_armv6-vfp-eabihf-Eval.tar.gz
+./tools/prepare_acontis_ecmaster.sh
+
```
Comment (pbruenn) [inline README.md]: If that download issue goes away, move this below the "make kernel" step

File: README.md [hunk 3]
```diff
@@ -54,5 +64,12 @@ Password: root
 
 Please change the root password immediately and additionally create your own user.
 
+### acontis EC-Master example (optional)
+To run the EcMasterDemo, extract the EC-Master SDK in /opt/EC-Master and start it from /opt/EC-Master/Bin/Linux/armv6-vfp-eabihf using:
+```
+EcMasterDemo -ccat 1 1
+```
+See manuals in the SDK's "Doc" folder for how to build and run EC-Master applications
+
```
Comment (pbruenn) [inline README.md:70]: provide a script for this

File: README.md (hunk 1 above)
Comment (PaulBussmann-acontis) [reply README.md]: I replaced EC-Master SDK evaluation registration by direct download link and moved instructions below the "make kernel" step with commit 844bbfbd49397d4e3375b0e8209a10c43dd37570

File: README.md (hunk 2 above)
Comment (PaulBussmann-acontis) [reply README.md]: I replaced EC-Master SDK evaluation registration by direct download link and moved instructions below the "make kernel" step with commit 844bbfbd49397d4e3375b0e8209a10c43dd37570

File: README.md (hunk 3 above)
Comment (PaulBussmann-acontis) [reply README.md:70]: I added acontis EC-Master SDK contents to /opt/EC-Master and added /usr/local/bin/RunEcMasterDemo.sh to RootFS with commit 6ae9fcd10db4b64971c105f6732d3716fc69a204

**Comment by PaulBussmann-acontis** [comment]
Thank you for reviewing the pull request to add acontis EC-Master EtherCAT support to Beckhoff CX9020!

I replaced EC-Master SDK evaluation registration by direct download link and moved instructions below the "make kernel" step with commit 844bbfbd49397d4e3375b0e8209a10c43dd37570
I added acontis EC-Master SDK contents to /opt/EC-Master and added /usr/local/bin/RunEcMasterDemo.sh to RootFS with commit 6ae9fcd10db4b64971c105f6732d3716fc69a204

Please integrate the changes to Beckhoff:master . Thank you very much!

**Comment by pbruenn** [comment]
We are currently adding a build and test pipeline on top of your pull request. Is it possible to extend your EcMasterDemo with some hardware interaction similar to the EtherLab example
In our test lab we use [CCAT|EL1004|EL1004|EL2004|EL2004|EK1110](https://github.com/Beckhoff/CX9020/blob/master/tests/etherlab/slaves.h) and just run a simple counter and "show" it on the EL2004 terminals.

**Comment by PaulBussmann-acontis** [comment]
You can flash the outputs of the EL2004 using „EcMasterDemo -ccat 1 1 -f <ENI-file> -flash <StationAddressOfEL2004>“.

**Pull request merged**